app/grandchallenge/uploads/widgets.py

```diff
@@ -36,3 +36,11 @@ def get_context(self, name, value, attrs):
         context = super().get_context(name, value, attrs)
         context["widget"]["attrs"]["multiple"] = True
         return context
+
+
+class DICOMUserUploadMultipleWidget(UserUploadMultipleWidget):
```

Comment:
Shouldn't this restrict `allowed_file_types`?

Reply:
It already does through the socket.